qiskit/dagcircuit/dagdependency.py

```diff
@@ -567,6 +568,12 @@ def merge_no_duplicates(*iterables):
             yield val
 
 
+@lru_cache(maxsize=None)
+def _get_identity(n):
```

Comment:
In general, I prefer avoiding the prefix `get_` when possible 
```suggestion
def _identity_matrix(n):
```